### ISSUE TITLE
Incorrect parsing for links/emojis in mrkdwn

### ISSUE THREAD

**Issue by StephenTangCook** [themashcodee/slack-blocks-to-jsx#38]
Mrkdwn text containing colons appear to be converting incorrect sections of the text into an emoji. 

Example:

```
[
  {
	"type": "context",
	"elements": [
		{
			"type": "mrkdwn",
			"text": "Pro-tip: This regex eagerly goes up to the https <https://app.slack.com/block-kit-builder|text> ."
		}
	]
},
  {
	"type": "section",
	"text": {
		"type": "mrkdwn",
		"text": "Pro-tip: This regex eagerly goes up to the https <https://app.slack.com/block-kit-builder|text> ."
	}
}
]
```

![Screenshot 2024-08-22 at 3 52 06 PM](https://github.com/user-attachments/assets/23a5096d-2873-4dc8-aa00-0dd48dfe3433)

**Comment by StephenTangCook** [comment]
Actually, it may be an issue with detecting links in mrkdwn. (Or it could be a separate issue). Have multiple links in the same mrkdwn text failed:
```
[
{
	"type": "section",
	"text": {
		"type": "mrkdwn",
		"text": "<https://google.com|one1> and <https://google.com|two1>"
	}
}
]
```
![Screenshot 2024-08-22 at 3 56 39 PM](https://github.com/user-attachments/assets/cf2f3d9f-ce50-4903-8dd3-8463efd1a93c)

**Comment by themashcodee** [comment]
@StephenTangCook resolved in [v0.5.5](https://github.com/themashcodee/slack-blocks-to-jsx/releases/tag/v0.5.5)
